applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_integration_schemes.cpp

```diff
@@ -32,6 +33,27 @@ std::unique_ptr<IntegrationScheme> MakeLobattoIntegrationScheme(std::size_t Numb
 
 std::vector<std::size_t> SupportedNumbersOfPointsForLobattoIntegration() { return {2, 3}; }
 
+std::unique_ptr<IntegrationScheme> MakeLumpedIntegrationScheme(std::size_t NumberOfPoints)
+{
+    return std::make_unique<LumpedIntegrationScheme>(NumberOfPoints);
+}
+
+std::vector<std::size_t> SupportedNumbersOfPointsForTriangleLumpedIntegration() { return {3, 6}; }
+
+std::vector<std::size_t> SupportedNumbersOfPointsForQuadrilateralLumpedIntegration()
+{
+    return {4, 8};
+}
+
+std::vector<std::size_t> SupportedNumbersOfPointsForLumpedIntegration()
+{
+    auto sup_triang_num = SupportedNumbersOfPointsForTriangleLumpedIntegration();
+    auto sup_quad_num   = SupportedNumbersOfPointsForQuadrilateralLumpedIntegration();
+    auto sup_num        = sup_triang_num;
+    std::move(sup_quad_num.begin(), sup_quad_num.end(), std::back_inserter(sup_num));
+    return sup_num;
+}
```

Comment:
Bit nitpicky, but this can be simplified slightly
```suggestion
std::vector<std::size_t> SupportedNumbersOfPointsForLumpedIntegration()
{
    auto result        = SupportedNumbersOfPointsForTriangleLumpedIntegration();
    auto sup_quad_num   = SupportedNumbersOfPointsForQuadrilateralLumpedIntegration();
    std::move(sup_quad_num.begin(), sup_quad_num.end(), std::back_inserter(result));
    return result;
}
```

Reply:
done.